src/workloads/sharding/BatchedUpdateOneWithoutShardKeyWithId.yml

```diff
@@ -0,0 +1,134 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  Runs the batched updateOne writes of type WithoutShardKeyWithId with a batch size of 1000.
+
+  The workload consists of 3 phases:
+    1. Creating an empty sharded collection distributed across all shards in the cluster.
+    2. Populating the sharded collection with data.
+    3. Running updateOne operations of type WriteWithoutShardKeyWithId.
+
+  The inserted documents have the following form:
+
+      {_id: 10, oldKey: 20, newKey: 30, counter: 0, padding: 'random string of bytes ...'}
+
+  The collection is sharded on {oldKey: 'hashed'}.
+
+GlobalDefaults:
+  - &Nop {Nop: true}
+
+  - &Database test
+  # Collection0 is the default collection populated by the MonotonicSingleLoader.
+  - &Collection Collection0
+  - &Namespace test.Collection0
+
+  # Note that the exact document size may exceed ApproxDocumentSize because of field names and other
+  # fields in the document.
+  - &ApproxDocumentSize 10000  # = 10kB
+  - &ApproxDocumentSize80Pct 8000  # = 8kB
+  - &DocumentCount 100000  # for an approximate total of 1GB
+
+  - &ShardKeyValueMin 1
+  - &ShardKeyValueMax 1000
+
+  - &Document {
```

Comment:
The Document anchor isn't used anywhere and is incomplete? Maybe fill out the rest of it and use it in the loader or remove it (you already explain the structure in the comment at the top).

Reply:
Removed.